docker/grouper/Dockerfile

```diff
@@ -0,0 +1,132 @@
+FROM jeanblanchard/java:jdk-7
```

Comment:
I'm guessing the answer is just "because it doesn't work with Java 8" but in case it isn't: is there a reason we're not just using our existing `discoenv/javabase` image which already has JCE etc. and should also have a JDK?